### PR TITLE
Fix for Carthage to Build

### PR DESCRIPTION
**Pull request by eSpecialized**
We found that tests were interfering with SQift builds using carthage, a small change allows them to build, and Xcode to test each Scheme

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 
 #### 5.x Releases
 
-- `5.0.x` Releases - [5.0.0](#500)
+- `5.0.x` Releases - [5.0.0](#500)||[5.0.1](#501)
```
Comment (cnoon) [inline CHANGELOG.md]: `[5.0.0](#500)||[5.0.1](#501)` -> `[5.0.0](#500) | [5.0.1](#501)`. It's just for markdown formatting.

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,6 +41,17 @@ All notable changes to this project will be documented in this file.
 - `0.2.x` Releases - [0.2.0](#020)
 - `0.1.x` Releases - [0.1.0](#010)
 
+## [5.0.1](https://github.com/Nike-Inc/SQift/releases/tag/5.0.1)
+
+Release on 2019-06-13.  All issues associated with this milestone can be found using this
```
Comment (cnoon) [inline CHANGELOG.md]: `Release on 2019-06-13.  All` -> `Released on 2019-06-27. All`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -41,6 +41,17 @@ All notable changes to this project will be documented in this file.
...
+[filter](https://github.com/Nike-Inc/SQift/milestone/12?closed=1).
+
+#### Updated
+
+-  Fixes Carthage build issues with failing tests. Updates to Scheme's allows carthage to do the build.
```
Comment (cnoon) [inline CHANGELOG.md]: How about?

```
- Fixed issue where Carthage was unable to build SQift due to schemes requiring test suite compilation.
  - Fixed by [William]...
```

File: CHANGELOG.md (hunk 1 above)
Comment (eSpecialized) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (eSpecialized) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (eSpecialized) [reply CHANGELOG.md]: Updated.

Comment (cnoon) [reply CHANGELOG.md]: This is still not quite right. Both should start with `Fixed` and typo in `was`. Second line is indented two spaces.

File: CHANGELOG.md (hunk 2 above)
Comment (cnoon) [reply CHANGELOG.md]: Still not correct. Should be `Released` and there's an extra space between `. All`.

File: CHANGELOG.md (hunk 3 above)
Comment (eSpecialized) [reply CHANGELOG.md]: Done

**Comment by cnoon** [comment]
Hey @eSpecialized, I've just merged a modified version of these changes in f88de59f. They'll go out shortly as part of the 5.0.1 release. Thank again for putting this together! 🍻

**Pull request closed without merging**